dashboard/modules/serve/serve_rest_api_impl.py

```diff
@@ -213,6 +215,41 @@ def validate_http_options(self, client, http_options):
                     f"updated: {divergent_http_options}."
                 )
 
+        def log_config_change_default_warning(self, config):
```

Comment:
We don't have to do it in this PR, but it would be nice to have a way to silence these warnings. If the only way for users to silence these warnings is to set the option explicitly, we should mention that in the log.

Reply:
Hmm, good point. It's a bit weird to say "to silence these warnings, set a value for max ongoing requests" though. Maybe we can have an environment variable for silencing these warnings? It's not exactly a deprecation warning though, so not sure if we can come up with a generic enough env var name.

Reply:
I think an env var would be nice. How about something like `RAY_SERVE_SILENCE_RAY231_WARNINGS`?

This is not blocking by the way. We can handle it in a follow up. I'll approve this PR.